lib/linux_admin/network_interface.rb

```diff
@@ -15,6 +15,8 @@ def self.dist_class(clear_cache = false)
       @dist_class ||= begin
         if [Distros.rhel, Distros.fedora].include?(Distros.local)
           NetworkInterfaceRH
+        elsif Distros.local == Distros.darwin
+          NetworkInterfaceDarwin
```

Comment:
I wonder if this is cleaner as:

```suggestion
        case Distros.local
        when Distros.rhel, Distros.fedora
          NetworkInterfaceRH
        when Distros.darwin
          NetworkInterfaceDarwin
```